### ISSUE TITLE
[3D visualization on Assembly]

### ISSUE THREAD

**Issue by florentgarin** [docdoku/docdoku-plm#791]
Display 3D visualization on permanent link also for assembly part.

**Comment by mguimard** [comment]
Should we also allow it for public and private shares ?

**Comment by florentgarin** [comment]
Let's think, what reason to not?

**Comment by mguimard** [comment]
This is quite complex, I can't find out a secure way to propagate the rights on a whole tree visibility. (This is essentially the case for public share.)

Moreover, the permalink shows the latest version of the requested part, which holds a mutable structure (its children can be modified and the tree can evolve). The access rights for each leaves must be calculated on the fly which could lead to performance issues on big product structures. 

I'll dig a bit more in the next days, but I feel that it should be implemented in two steps : 
* Show 3D assembly for authenticated users
* show 3D assembly for non authenticated users (private and public shares)

**Comment by florentgarin** [comment]
right, let's do that "Show 3D assembly for authenticated users" for release 2.0 we will see after.
